### ISSUE TITLE
Add HbA1c Entry

### ISSUE THREAD

**Issue by bkerensa** [Glucosio/glucosio-android#38]
We need to add support to accept HbA1c readings to the DB and in the add recording UI

**Comment by TimothyCuffe** [comment]
I've noticed a small issue todo with this feature and since this is still marked open I'm trying to avoid duplicates.
This is my first report on this repository so I've followed the bug reporting template since that's what your documentation states,
but all it basically boils down to is that HBA1C has been implemented as HB1AC.
As far as I can tell on google HB1AC only seems to be used in typos.

Steps to reproduce:
1. Tap the add reading + button
2. Read the options

Expected behaviour:
ability to add HBA1C reading

Actual behaviour:
ability to add HB1AC reading

Locale: English (UK)
Measurements: mmol/L
Using backups: No

Device configuration
Device: A0001
OS: Cyanogen
Version: 12.1.1
Downloaded app from: Google play store

Thanks,
            Timothy.
